llvm/lib/Support/VirtualFileSystem.cpp

```diff
@@ -302,6 +305,17 @@ class RealFileSystem : public FileSystem {
     return Storage;
   }
 
+  ErrorOr<std::unique_ptr<File>> openFileForRead(const Twine &Name,
```

Comment:
Thanks, Could you mark this as static.

Reply:
I don't think I'm able to mark it static because it calls adjustPath which is non-static

Reply:
np.  I was misreading the diff.  I didn't see it was a private member function.  It looked like a file scope function.  I was thinking file scope static, not static member.